api/src/main/java/javax/mvc/engine/ViewEngineContext.java

```diff
@@ -74,21 +71,6 @@
     Models getModels();
 
     /**
-     * Returns HTTP request object from the Servlet container.
-     *
-     * @return HTTP request object.
-     */
-    HttpServletRequest getRequest();
-
-    /**
-     * Returns HTTP response object from the servlet container. The underlying
-     * output stream should be used to write the result of processing a view.
-     *
-     * @return HTTP response object.
-     */
-    HttpServletResponse getResponse();
-
```

Comment:
@lefloh: Sorry, I think my post wasn't clear enough. With "No `@Produces`" I actually was thinking of "No explicit `@Produces`. Of cause `text/html` (without encoding) is the default.

And I agree with you that it is weird that Ozark writes the `Content-Type` header. This should be done by the view engine. And in this case the view engine would either use the media type implicitly or explicitly specified via `@Consumes` or whatever it finds in the template.